app/assets/javascripts/components/event_invites.js.jsx

```diff
@@ -0,0 +1,84 @@
+var EventInvites = React.createClass({
+  propTypes: {
+    users: React.PropTypes.array,
+    event_id: React.PropTypes.number
+  },
+
+  getInitialState: function () {
+    return {
+      filterQuery: '',
+      user_id: '',
+      message: ''
+    };
+  },
+
+  handleSendInvite: function () {
+    var that = this;
+    $.ajax({
+      method: "POST",
+      url: "/invites",
+      data: {
+        invite: {
+          user_id: this.state.user_id,
+          event_id: this.props.event_id
+        }
+      }
+    }).done(function(response){
+      console.log(response)
```

Comment:
We should probably remove any console log statement before a PR to clean up our js console output.